lib/generators/init-generator.js

```diff
@@ -339,8 +338,7 @@ module.exports = class InitGenerator extends Generator {
 				if (regExpForStyles) {
 					if (this.isProd) {
 						this.configuration.config.topScope.push(tooltip.cssPlugin());
-						// TODO: Replace with regular version once v.4 is out
-						this.dependencies.push("extract-text-webpack-plugin@next");
+						this.dependencies.push("extract-text-webpack-plugin");
```

Comment:
We'll still need to use the pre-release version.

Reply:
yup just pushed smth to reference issue

Reply:
will work on it tonight :)